theme/debugger.less

```diff
@@ -63,17 +63,33 @@
     }
 
     .variable {
-        color: white;
+    }
+    .transparent{
+        opacity: 0;
     }
     .variable.changed {
-        outline: solid 2px yellow;
```

Comment:
Just delete the empty less rules; vscode complains about them

Reply:
Sure.